distributed/tests/py3_test_asyncio.py

```diff
@@ -12,11 +12,11 @@
 from distributed.utils_test import slowinc
 
 from distributed.asyncio import AioClient, AioFuture, as_completed, wait
-from distributed.utils_test import inc, div, loop
+from distributed.utils_test import inc, div
 
 
-async def test_asyncio_start_shutdown(loop):
-    c = AioClient(loop=loop, processes=False)
+async def test_asyncio_start_shutdown():
```

Comment:
AioClient no longer takes a "loop" parameter, it just uses the current asyncio event loop.